gcloud/storage/blob.py

```diff
@@ -150,9 +150,9 @@ def public_url(self):
             quoted_name=quote(self.name, safe=''))
 
     def generate_signed_url(self, expiration, method='GET',
-                            client=None, credentials=None,
-                            response_type=None, response_disposition=None,
-                            generation=None):
+                            content_type='application/octet-stream',
```

Comment:
Oh, None would be better. Will push another commit.

I was considering one specific case for POSTing tp signed URI. I realized
that I cannot assume HTTP method and content type, since it will bother use
case for GET or other methods.

2016년 2월 13일 (토) 오후 4:27, Danny Hermes <notifications@github.com>님이 작성:

> In gcloud/storage/blob.py
> <https://github.com/GoogleCloudPlatform/gcloud-python/pull/1447#discussion_r52824692>
> :
>
> > @@ -150,9 +150,9 @@ def public_url(self):
> >              quoted_name=quote(self.name, safe=''))
> >
> >      def generate_signed_url(self, expiration, method='GET',
> > -                            client=None, credentials=None,
> > -                            response_type=None, response_disposition=None,
> > -                            generation=None):
> > +                            content_type='application/octet-stream',
>
> RE: application/octet-stream vs. None. Why does it prevent undefined
> behavior? The method being called defines what happens when None is
> passed in.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-python/pull/1447/files#r52824692>
> .
>